weave

```diff
@@ -48,9 +48,11 @@ usage() {
     echo "weave reset"
     echo "weave rmpeer       <peer_id>"
     echo
-    echo "where <peer>    is of the form <ip_address_or_fqdn>[:<port>], and"
-    echo "      <cidr>    is of the form <ip_address>/<routing_prefix_length>"
-    echo "      <peer_id> is a <nickname> or weave internal peer ID"
+    echo "where <peer>         is of the form <ip_address_or_fqdn>[:<port>], and"
+    echo "      <cidr>         is of the form <ip_address>/<routing_prefix_length>"
+    echo "      <peer_id>      is a <nickname> or weave internal peer ID"
+    echo "      <address_spec> is either a single address in <cidr> format or"
+    echo "         --subnet <cidr> or --subnet default to fetch from the allocator"
```

Comment:
Following my [commit just now](https://github.com/weaveworks/weave/commit/2d701972211129ed6246d9d7b523a20dfb805e2d) this should be `= <cidr> | --subnet <cidr> | --subnet default`

No need to get more descriptive.